pkg/client/chartmuseum/chartmuseum_test.go

```diff
@@ -0,0 +1,233 @@
+package chartmuseum
```

Comment:
please use `chartmuseum_test` and import the `chartmuseum` package here

Reply:
done